src/Website/Core/BrewClubViewPage.cs

```diff
@@ -17,16 +18,25 @@ public SiteLocalizer Localizer
 			get { return (SiteLocalizer)ViewBag.Localizer; }
 		}
 
-		public string PageTitle { get; set; }
+		public string PageTitle
+		{
+			get { return (string)ViewBag.PageTitle; }
+			set { ViewBag.PageTitle = value; }
+		}
```

Comment:
Apparently the old code didn't actually work.  I suspect that the various partial views create new instances of the base class, which causes the PageTitle to get lost from page to page.  Wrapping the property around the ViewBag avoids that.